reservedip/Readme.md

```diff
@@ -0,0 +1,23 @@
+# Snippet - Supported Reserved IP Use Cases
+
+This template demonstrates the currently supported use case for Reserved IP.  A Reserved IP is simply a statically allocated Public IP.  
+
+There is currently only one supported use case: assign a Reserved IP to the front end of the Azure Load Balancer.  
+
+This template goes one step further: it both creates a new Reserved IP and assigns it to a load balancer and it uses a previously existing Reserved IP and assigns it to a separate load balancer.
```

Comment:
After the execution, will the user have 1 new LB and 1 new PIP and the LB will have two PIPs?